draft-ietf-rats-architecture.md

```diff
@@ -130,43 +124,44 @@ Appraisal Policy for Attestation Result:
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the Evidence it has appraised
```

Comment:
I'm not sure the change is technically correct.  The verifier vouches for the level of trustworthiness of the attester, and it vouches for the results it signs.  But it could (for example) ignore extra claims in the evidence (e.g., GPS location) that it cannot vouch for, and provide only vouched-for claims in the results.  Hence it need not vouch for the validity of the entire evidence.   I recommend reverting.

Reply:
I agree with Dave here. And I even think that we should word-smith a proposed change out of this comment and be more verbose here (yes, I know... even more words, but these seem to be universal useful).